src_kt/javatests/arcs/crdt/entity/CrdtEntityDescriptorTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.crdt.entity
+
+import org.junit.BeforeClass
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [CrdtEntityDescriptor]. */
+@RunWith(JUnit4::class)
+class CrdtEntityDescriptorTest {
+  @Test(expected = IllegalArgumentException::class)
+  fun unsupportedType_triggersIllegalArgumentException() {
+    CrdtEntityDescriptor("dummy" to MyDummyType::class)
```

Comment:
In Java we tend to prefer using `assertThrows` to `expected`, I assume the same is true for tests in Kotlin?
```
assertThrows({ CrdtEntityDescriptor("dummy" to MyDummyType::class) }, IllegalArgumentException::class)
```

Reply:
For some reason assertThrows isn't in the classpath... I'll try again.